Ansible/ansible_collections/jfrog/platform/roles/artifactory_nginx/tasks/Debian-ownstream.yml

```diff
@@ -0,0 +1,25 @@
+- name: ensure apt-transport-https is installed
+  anible.builtin.napt:
+    name: apt-transport-https
+    state: present
+
+- name: Add upstream nginx apt key
+  become: true
+  ansible.builtin.apt_key:
```

Comment:
@bbaassssiiee  Debian-ownstream.yml and Debian-upstream.yml seems to be same , can we have a single file (with nginx_upstream_repo_baseurl can be changed based on which stream) we use ?

Reply:
`upstream` trusts Nginx more than Debian or Red Hat because of faster fixes, `ownstream` is easier in air-gapped environments. I think it is best to leave the choice to the user. Maybe in another PR this can be refactored to use clever variables to allow both use-cases with minimal code.